virttest/virsh.py

```diff
@@ -1830,21 +1830,23 @@ def attach_disk(name, source, target, extra="", **dargs):
     return command(cmd, **dargs)
 
 
-def detach_disk(name, target, extra="", wait_remove_event=False, event_timeout=7, **dargs):
+@EventTracker.wait_event
+def detach_disk(name, target, extra="", wait_for_event=False,
+                event_type="device-removed", event_timeout=7, **dargs):
```

Comment:
Some feedback practices from previous other event value, maybe need extend event_timeout to a little longer value, such as event_timeout=10

Reply:
done